cgi-bin/DW/Controller/Talk.pm

```diff
@@ -8,6 +8,154 @@ use Carp;
 
 DW::Routing->register_string( '/talkpost_do', \&talkpost_do_handler, app => 1 );
 
+# I think this canon isn't written down anywhere else, so HWAET: I sing a record
+# of every damn form field that comes through in reply form POSTs. -NF
```

Comment:
oh my lord this comment T_T

this is why we can't have nice things. or, alternately: this is why the world is moving towards static typing as a necessity for building software at anything beyond toy-scale.